helpers/utils.js

```diff
@@ -1,18 +1,29 @@
-const getTestUserLoginDetails = () => {
-    if (!process.env.CCD_E2E_PASSWORD) {
-        throw new Error('You need to set CCD_E2E_PASSWORD env variable');
+const getSolicitorLoginDetails = () => {
+    if (!process.env.CCD_E2E_EMAIL || !process.env.CCD_E2E_PASSWORD) {
```

Comment:
So in the other `div` projects we are using the infrastructure files to inject environment variables, and applying them to a `conifg.yml` file to be accessed in application code. You can see an example of how the infrastructure files pull in `var`s from the Azure vault [here](https://github.com/hmcts/div-respondent-frontend/blob/master/infrastructure/main.tf):
```
IDAM_API_URL = "${var.idam_api_url}"
```
And then in the `config/default.yml`:
```
services:
    idam:
        apiUrl: IDAM_API_URL
```

Then we can use the npm module called `[config](https://github.com/lorenwest/node-config/wiki/Configuration-Files)` to read the yaml and use it application code. 

Doing all this means we don't need to maintain environment variables in the various pipelines this application may run in. We can also utilise the Azure vault for secrets and so on.

Reply:
Thanks @alexwakeman , as discussed I'll need your help to do this in the next story. Appreciate